### PR TITLE
Feature/semantic accessibility

### PR DESCRIPTION
**Pull request by juancho11gm**

### CODE REVIEW

File: html/semantic/structure.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>My page title</title>
+    <link rel="stylesheet" href="style.css">
```
Comment (fabianmarinog) [inline html/semantic/structure.html]: This will throw an error if `style.css` doesn't exist. Check my suggestion below.

Comment (fabianmarinog) [inline html/semantic/structure.html]:
```suggestion
    <link rel="stylesheet">
```

File: git/juansebastian.html [hunk 2]
```diff
@@ -2,9 +2,38 @@
...
+      </article>
+      <aside>
+        <ul>
+          <li>
+              <address>Calle 51A#5-22</address>
+          </li>
+          <li>
+              <p>This file was created at <time>17:40</time> .</p>      
+          </li>
+        </ul>
```
Comment (fabianmarinog) [inline git/juansebastian.html]: This info doesn't need to be in a list.

File: html/semantic/structure.html (hunk 1 above)
Comment (juancho11gm) [reply html/semantic/structure.html]: Done!

File: git/juansebastian.html (hunk 2 above)
Comment (juancho11gm) [reply git/juansebastian.html]: I've changed it.

**Pull request merged**